askama_axum/Cargo.toml

```diff
@@ -15,12 +15,13 @@ readme = "README.md"
 
 [dependencies]
 askama = { version = "0.12", path = "../askama", default-features = false, features = ["with-axum", "mime", "mime_guess"] }
-axum-core = "0.3"
-http = "0.2"
+axum-core = "0.4"
+http = "1.0"
 
 [dev-dependencies]
-axum = { version = "0.6", default-features = false }
-hyper = { version = "0.14", features = ["full"] }
+axum = { version = "0.7", default-features = false }
+hyper = { version = "1.0", features = ["full"] }
+http-body-util = "0.1"
```

Comment:
Please add this before hyper so that the dependencies remain in alphabetic order.